package.json

```diff
@@ -289,8 +290,18 @@
           "ansible"
         ],
         "extensions": [
-          ".yml",
-          ".yaml"
+          ".ansible.yml",
+          ".ansible.yaml"
+        ],
+        "filenamePatterns": [
+          "**/playbooks/*.yml",
+          "**/playbooks/*.yaml"
+        ],
+        "filenames": [
+          "site.yml",
+          "site.yaml",
+          "playbook.yml",
```

Comment:
In linter we use `*playbook*/yml` pattern as being a little bit more flexible. See https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/config.py#L31-L32

Reply:
Thanks for the suggestion. I can include this!

Reply:
updated.